WordPress/Classes/Utility/ImmuTable.swift

```diff
@@ -279,6 +279,14 @@ public class ImmuTableViewHandler: NSObject, UITableViewDataSource, UITableViewD
         return cell
     }
 
+    public func tableView(tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return viewModel.sections[section].headerText
+    }
+
+    public func tableView(tableView: UITableView, titleForFooterInSection section: Int) -> String? {
+        return viewModel.sections[section].footerText
```

Comment:
Getting rid of custom header/footer class in #4591. I'd say let's merge this first, since the current API promises to show a header/footer text and does nothing, and I'll handle the customisation on another PR once #4591 is merged.